peaq-did-proto-js.d.ts

```diff
@@ -87,7 +87,7 @@ declare module 'peaq-did-proto-js' {
     clearVerificationmethodsList(): this;
 
     getSignature(): Signature | undefined;
-    setSignature(value?: Signature): this;
+    setSignature(value?: Signature): this; // do we want a list of signatures? will have to change proto if we do
```

Comment:
Will we only ever have 1 siganature? When setting the verifications and services it is able to add multiple. However, for signature this is not possible and it would be the proto document would be to be changes so setSignature would be: `setSignature(value?: Signature[]): this;`

Similar to the `setServicesList(value:Service[]): this;`

Reply:
single signture